src/Site_WP_Command.php

```diff
@@ -330,6 +330,15 @@ private function configure_site_files() {
 			$this->fs->dumpFile( $site_nginx_default_conf, $default_conf_content );
 			$this->fs->mkdir( $site_conf_dir . '/php-fpm' );
 			$this->fs->dumpFile( $site_php_ini, $php_ini_content );
+			$this->fs->mkdir( $site_conf_dir . '/postfix' );
```

Comment:
Postfix is a utility other packages will also require. It is best if the code to configure it goes inside `site-utils` of EasyEngine core. This block of code could become something like: `EE\SiteUtils\configure_postix($site_conf_dir, $site_name)` in core so that it can be easily re-used in other packages.

Reply:
@mrrobot47 I am not really sure what you mean by _Postfix is a utility other packages will also require_.

Also, how are you comparing the following to just creating a directory to store postfix's configuration?
```
EE\SiteUtils\configure_postix($site_conf_dir, $site_name)
```